views/project/user.erb

```diff
@@ -151,4 +151,44 @@
       </div>
     </form>
   </div>
+  <!-- Access Policy -->
+  <div id="policy-editor">
```

Comment:
What do you think about making this collapsible and collapsed by default?

Reply:
It makes sense. However, I'm considering delaying it. Given that our current users have only advanced policies for their projects, we can keep editor visible until we migrate them to managed policies. Otherwise, not being able to see their current policy by default scares them

Reply:
That is fine by me.